<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGameFiles</class>
 <widget class="QDialog" name="AddGameFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление файлов игры</string>
  </property>
  <widget class="QLineEdit" name="le_GamePath">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Game">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название игры:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_GamePath">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выберите файл игры</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Save">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Browse">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_Game">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
